### PR TITLE
A few SWAT changes

### PR DESCRIPTION
**Pull request by Bawhoppen**
:cl:
add: You can now order SWAT vest alternatively to SWAT suits at cargo for the same price.
tweak: SWAT armor is now better than sec armor again.
tweak: Simple Mob NPCs that have SWAT helms now have SWAT vests to go with instead of regular armor.
/:cl:

Changelog explains it all.

### CODE REVIEW

File: _maps/shuttles/whiteship_meta.dmm [hunk 1]
```diff
@@ -989,9 +989,9 @@
 	},
 /obj/item/clothing/gloves/color/black,
 /obj/item/clothing/gloves/color/black,
-/obj/item/clothing/suit/armor/vest,
-/obj/item/clothing/suit/armor/vest,
 /obj/structure/table,
+/obj/item/clothing/suit/armor/swat,
+/obj/item/clothing/suit/armor/swat,
```
Comment (Niknakflak) [inline _maps/shuttles/whiteship_meta.dmm]: looks like an undocumented change to me

**Comment by optimumtact** [comment]
You have changed very little about this PR from #19539 which everyone hated.

I still hate this one too

**Comment by Bawhoppen** [comment]
everyone hated that PR because of the sec changes, this isn't a sec change

File: _maps/shuttles/whiteship_meta.dmm (hunk 1 above)
Comment (Bawhoppen) [inline _maps/shuttles/whiteship_meta.dmm]: not really? I changed it so the armor that was to go with the SWAT helm had the adequate vest armor

**Comment by optimumtact** [comment]
Kor did this to you.

File: code/modules/mob/living/simple_animal/corpse.dm [hunk 2]
```diff
@@ -11,7 +11,7 @@
 /obj/effect/mob_spawn/human/corpse/syndicatesoldier
 	name = "Syndicate Operative"
 	uniform = /obj/item/clothing/under/syndicate
-	suit = /obj/item/clothing/suit/armor/vest
+	suit = /obj/item/clothing/suit/armor/swat
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/corpse.dm]: why

File: code/modules/mob/living/simple_animal/corpse.dm [hunk 3]
```diff
@@ -118,7 +118,7 @@
 /obj/effect/mob_spawn/human/corpse/nanotrasensoldier
 	name = "Nanotrasen Private Security Officer"
 	uniform = /obj/item/clothing/under/rank/security
-	suit = /obj/item/clothing/suit/armor/vest
+	suit = /obj/item/clothing/suit/armor/swat
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/corpse.dm]: why

File: code/modules/mob/living/simple_animal/corpse.dm (hunk 2 above)
Comment (Bawhoppen) [inline code/modules/mob/living/simple_animal/corpse.dm]: they have SWAT helms but regular armor and it has ALWAYS bothered me

Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/corpse.dm]: did you update the mob sprites?

Comment (Bawhoppen) [inline code/modules/mob/living/simple_animal/corpse.dm]: nah but it looks pretty similar anyways the sprites are out of date anyways I can if its really critical

**Comment by Bawhoppen** [comment]
okay I have updated the mob sprites and readded a 4-dir caucasian human base for future use

File: code/modules/awaymissions/corpse.dm [hunk 4]
```diff
@@ -249,7 +249,7 @@
 /obj/effect/mob_spawn/human/syndicatesoldier
 	name = "Syndicate Operative"
 	uniform = /obj/item/clothing/under/syndicate
-	suit = /obj/item/clothing/suit/armor/vest
+	suit = /obj/item/clothing/suit/armor/swat
```
Comment (optimumtact) [inline code/modules/awaymissions/corpse.dm]: stop

File: code/modules/awaymissions/corpse.dm [hunk 5]
```diff
@@ -450,7 +450,7 @@
 /obj/effect/mob_spawn/human/nanotrasensoldier
 	name = "Nanotrasen Private Security Officer"
 	uniform = /obj/item/clothing/under/rank/security
-	suit = /obj/item/clothing/suit/armor/vest
+	suit = /obj/item/clothing/suit/armor/swat
```
Comment (optimumtact) [inline code/modules/awaymissions/corpse.dm]: just stop you're going to get your pr closed for having too many changes.

Comment (Bawhoppen) [inline code/modules/awaymissions/corpse.dm]: literally what are you talking about? its just another variant of the other corpse in a different file

**Comment by Bawhoppen** [comment]
got a new tacticool sprite for it 
ss 2016-08-09 at 09 46 37

**Comment by Chocobro** [comment]
nice :+1:

**Comment by Shadowlight213** [comment]
This is kinda ridiculous. As oranges pointed out, this is better than the captain's armor in al.oat every way. Plus the vests have no slowdown that I can see and have the same armor values.
Also, I checked their values versus the syndie hardsuits. 
They are better than standard syndie hardsuits in melee, laser, and energy. Only 10 less in bullet.
Another fun fact! They have only 10 less energy (stun) resist than the reflector vest! Elite syndie suits only have 25.

**Comment by Cheridan** [comment]
Yeah, these aren't getting into cargo.

**Comment by KorPhaeron** [comment]
well okay then, headcoder said no

**Comment by Bawhoppen** [comment]
@Shadowlight213 Why wouldn't it be better than captain's armor?

Why are you guys blaming me for what Steelpoint did? I didn't want Steelpoint to buff standard armor.
SWAT gear would logically make sense to be better than fucking shitty mallcop armor.

What do you want me to remove all SWAT gear from cargo for this PR to get merged?

**Comment by optimumtact** [comment]
If anything nerf sec armour.

**Comment by Bawhoppen** [comment]
WILL THAT GET MERGED? IF SO I WILL GLADLY.

**Comment by Shadowlight213** [comment]
Just remove it from cargo.

**Comment by Bawhoppen** [comment]
I don't want to remove it entirely,
Also Shadowlight, I believe that the energy in the armor var only affects disablers, the deflect and reflection is an ablative snowflake thing

**Comment by Shadowlight213** [comment]
From what I was aware, energy resist has an effect on stun times from at least tasers

**Comment by KorPhaeron** [comment]
why does this keep getting closed/opened

**Comment by optimumtact** [comment]
Because I hate it and the other maintainers are soft and I'm bound by bias against acting on this.

**Comment by Jordie0608** [comment]
It's ok I don't like it either.

**Pull request closed without merging**